website/docusaurus.config.js

```diff
@@ -112,6 +112,7 @@ const config = {
     description:
       'An optimized site generator in React. Docusaurus helps you to move fast and write content. Build documentation websites, blogs, marketing pages, and more.',
   },
+  staticDirectories: ['static', '_dogfooding/_asset-tests'],
```

Comment:
suggest to use one absolute path here for `'_dogfooding/_asset-tests'` => ensures it works fine with relative or absolute